internal/core/state/api/store.josh.hcl

```diff
@@ -62,6 +62,11 @@ interface "store" {
     input "id" "string" {}
   }
 
+  method "ensure-device" {
+    doc = "Ensure that one-time device initialization has been completed. This is safe to call whenever the server is restarted."
```

Comment:
are there times when this is not safe to call? or do you just mean it is safe to call multiple times?